vmaas_sync/system_culling.go

```diff
@@ -0,0 +1,17 @@
+package vmaas_sync //nolint:golint,stylecheck
+
+import (
+	"app/base/database"
+	"time"
+)
+
+func RunSystemCulling() {
+	ticker := time.NewTicker(time.Hour)
```

Comment:
This should be rather implemented as a job. We shouldn't mix different tasks in one microservice.